loop/index.js

```diff
@@ -42,6 +42,8 @@ var storage = getStorage(conf.get("storage"), {
 var tokBox = new TokBox(conf.get('tokBox'));
 
 var ravenClient = new raven.Client(conf.get('sentryDSN'));
+ravenClient.captureError(new Error('Server could communicate with Sentry'));
```

Comment:
Good news, instantiating a new client does not seem to be necessary:
https://github.com/getsentry/raven-node/pull/8